app/tests/src/evaluation/test_literalai_exporter.py

```diff
@@ -47,19 +48,18 @@ def append_dangling_step(thread: Thread):
 def test_convert_to_qa_rows_and_save_csv(caplog):
     threads = create_threads(3)
     dangling_step = append_dangling_step(threads[1])
-    project_id = "Test_Project_1234ABC"
     with caplog.at_level(logging.INFO):
-        qa_rows = convert_to_qa_rows(project_id, threads)
+        qa_rows = convert_to_qa_rows("Test_Project_1234ABC", threads)
     assert len(qa_rows) == 3
     assert f"Ignoring dangling step {dangling_step.id!r}"
```

Comment:
Check if dangling step id is in log message fixture

```suggestion
    assert f"Ignoring dangling step {dangling_step.id!r}" in caplog.text
```